css/datagrid.css

```diff
@@ -129,9 +129,9 @@
 }
 
 .hc-dg-cell-input {
-    width: calc(100% + 16px);
-    margin: 0 -8px;
-    padding: 0 6px;
+    width: calc(100% + 24px);
```

Comment:
Could we replace this `width: calc(...)` with `flex: 1`?

Reply:
Not really, it causes problems with width and highlighting input.